source/moto_payments/index.html.md.erb

```diff
@@ -16,35 +16,67 @@ This means you can take payments:
 
 These payments are also known as Mail Order / Telephone Order (MOTO) payments.
 
-You cannot take MOTO payments if your payment service provider (PSP) is SmartPay.
+You cannot take MOTO payments:
 
-You also cannot take MOTO payments through a [payment link](https://www.payments.service.gov.uk/govuk-payment-pages/) - [contact us](/support_contact_and_more_information/#contact-us) if you'd like us to develop this feature.
+- if your payment service provider (PSP) is Smartpay
+- through a [payment link](https://www.payments.service.gov.uk/govuk-payment-pages/)
 
-## If your PSP is Worldpay or ePDQ
+Our product roadmap is based on feedback. [Contact us](/support_contact_and_more_information/#contact-us) if you'd like us to investigate taking MOTO payments through payment links.
 
-If your PSP is Worldpay or ePDQ, you must first:
+## Set up MOTO payments
 
-- [create a new GOV.UK Pay service](https://selfservice.payments.service.gov.uk/create-service/register) for MOTO payments
-- contact your PSP and ask for a new merchant code for MOTO payments
+You can set up MOTO payments on either a test account or a live account.
 
-[Contact Government Banking](https://www.gov.uk/government/groups/government-banking-service-gbs) instead if you use them.
+MOTO payment screens are the same as online payment screens, excluding the following:
 
-## Set up your account
+- MOTO payments screens do not have a section for billing address
+- [Strong Customer Authentication](/set_up_3dsecure/#set-up-strong-customer-authentication-3ds1-and-3ds2) does not apply to MOTO payments
```

Comment:
should we link to the product pages or the docs here? Product pgs are more about what SCA is. but maybe makes sense if docs link to docs.

Reply:
I'd prefer to keep within the docs in case users are following the process and don't want to jump out of the docs.

Reply:
Doesn't that page have the SCA definition at the beginning? The first words on https://docs.payments.service.gov.uk/set_up_3dsecure/#set-up-strong-customer-authentication-3ds1-and-3ds2 are "Strong Customer Authentication (SCA) makes online payments more secure for users and reduces fraud. SCA is part of the Payment Service Directive 2 (PSD2)." Or am I misunderstanding your comment?

Reply:
well, it is supporting information for the task rather than the task itself or an intro to the task, and content best practice is to put that supporting info at the end